site/public/js/calendar.js

```diff
@@ -396,6 +454,132 @@ function generateCalendarOfMonth(view_year, view_month) {
     return table;
 }
 
+/**
+ * This function creates a table that shows the calendar for one week.
+ *
+ * @param view_year : int year that the calendar is viewing
+ * @param view_month : int month that the calendar is viewing (1 as January and 12 as December)
+ * @param view_day : int day that the calendar is viewing
+ * @returns {string} the HTML string contains the entire calendar table displaying view_month/view_year
```

Comment:
Doesn't this function actually return an `HTMLElement`, similar to the function above it?